test/remote/gateways/remote_commerce_hub_test.rb

```diff
@@ -109,7 +117,7 @@ def test_successful_purchase_with_stored_credential_framework
   def test_failed_purchase
     response = @gateway.purchase(@amount, @declined_card, @options)
     assert_failure response
-    assert_equal 'Unable to assign card to brand: Invalid.', response.message
```

Comment:
⚠️ Warning:

Seems that we need to assert with `assert_match` on this, seems that depending on credentials/features available this message has that final `.`

Reply:
Done!